packages/vm/src/buildBlock.ts

```diff
@@ -228,7 +227,7 @@ export class BlockBuilder {
     const transactionsTrie = await this.transactionsTrie()
     const receiptTrie = await this.receiptTrie()
     const logsBloom = this.logsBloom()
-    const gasUsed = this.gasUsed
+    const gasUsed = new BN(this.gasUsed.toString(10), 10)
```

Comment:
would suggest leaving the default values out when unneeded, i.e.
```suggestion
    const gasUsed = new BN(this.gasUsed.toString())
```